phys2bids/physio_obj.py

```diff
@@ -250,6 +251,8 @@ def check_trigger_amount(self, chtrig=1, thr=2.5, num_timepoints_expected=0, tr=
         Notes
         -----
         Outcome:
+         self.thr:
+            Threshold used by the function to detect trigger points.
```

Comment:
self.thr already in the docstring

Reply:
Great! Can you add it to the docstring of the _class_ now?
It should go around [this line](https://github.com/physiopy/phys2bids/blob/386f3c2c51bf8774cb3dc91ad077290c90fa058d/phys2bids/physio_obj.py#L108), after the `num_timepoints_found` Attribute, that is computed in the same function.